lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorException.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using Org.Apache.REEF.Driver.Task;
+using Org.Apache.REEF.Utilities;
+
+namespace Org.Apache.REEF.Driver.Evaluator
+{
+    // Public only such that it can be used in the bridge.
+    /// <summary>
+    /// </summary>
+    public sealed class EvaluatorException : Exception, IIdentifiable
+    {
+        private readonly string _evaluatorId;
+        private readonly IRunningTask _runningTask;
+
+        internal EvaluatorException(string evaluatorId)
+        {
+            _evaluatorId = evaluatorId;
+            _runningTask = null;
+        }
+
+        internal EvaluatorException(string evaluatorId, string message, Exception cause)
```

Comment:
Good call. I copied the code from the old instance of this type. I fixed it to use default parameters instead.